bungiesearch/management/commands/search_index.py

```diff
@@ -132,9 +132,11 @@ def handle(self, *args, **options):
                 mapping = {}
                 for mdl_idx in src.get_model_indices(index):
                     mapping[mdl_idx.get_model().__name__] = mdl_idx.get_mapping()
-
+                
+                analysis = mdl_idx.collect_analysis()
```

Comment:
Just associating the mapping with the model name if there are multiple model indices. I've tried associating analysis with model name, and create indices for each model separately - both yield parser exception errors.